pages/place/content/fields.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
   changesToRestrictions: {
     label: 'Changes to restrictions',
     conditionalReveal: {
-      label: 'Do these restrictions need to be changed?',
+      label: 'Does this area\'s restrictions need to be changed?',
```

Comment:
Should this be "Do this area's restrictions need to be changed?" since the restrictions are plural?

Reply:
I should have changed this now